README.Rmd

```diff
@@ -129,11 +129,12 @@ Each `xportr_` function has been written in a way to take in a part of the speci
 
 ```{r, message=FALSE, eval=FALSE}
 adsl %>%
-  xportr_type(var_spec, "ADSL") %>%
-  xportr_length(var_spec, "ADSL") %>%
-  xportr_label(var_spec, "ADSL") %>%
-  xportr_order(var_spec, "ADSL") %>%
-  xportr_format(var_spec, "ADSL") %>%
+  xportr_metadata(var_spec, "ADSL") %>%
+  xportr_type() %>%
+  xportr_length() %>%
+  xportr_label() %>%
+  xportr_order() %>%
+  xportr_format() %>%
   xportr_write("adsl.xpt", label = "Subject-Level Analysis Dataset")
```

Comment:
@atorus-research/xportr-development-team I'm wondering is there a way to have xportr_write take information from the spec files and apply the dataset label and create the `.xpt` based on the name of the final dataset file with `xportr_metadata`.  Looks so clean like below.  If so, I think this shoudl go into release 0.4

```
adsl %>%
  xportr_metadata(var_spec, "ADSL") %>%
  xportr_type() %>%
  xportr_length() %>%
  xportr_label() %>%
  xportr_order() %>%
  xportr_format() %>%
  xportr_write()
```

Reply:
AH man --- now I'm thinking ... what an `xportr_appply_all()` that takes all 6 functions and does it all in one swoop??

Reply:
@bms63 We do have xportr_df_label for the dataset label. I think updating the file to <_df_arg_>.xpt would be a nice default. `xportr_metadata_apply` would also be a nice add.